build/XUnit.props

```diff
@@ -1,13 +1,14 @@
 ﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <PackageReference Include="xunit" Version="2.2.0" />
+    <PackageReference Include="xunit" Version="2.3.0-beta5-build3769" />
     <PackageReference Include="xunit.abstractions" Version="2.0.1" />
-    <PackageReference Include="xunit.assert" Version="2.2.0" />
-    <PackageReference Include="xunit.core" Version="2.2.0" />
-    <PackageReference Include="xunit.extensibility.core" Version="2.2.0" />
-    <PackageReference Include="xunit.extensibility.execution" Version="2.2.0" />
-    <PackageReference Include="xunit.runner.console" Version="2.2.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+    <PackageReference Include="xunit.assert" Version="2.3.0-beta5-build3769" />
+    <PackageReference Include="xunit.core" Version="2.3.0-beta5-build3769" />
+    <PackageReference Include="xunit.extensibility.core" Version="2.3.0-beta5-build3769" />
+    <PackageReference Include="xunit.extensibility.execution" Version="2.3.0-beta5-build3769" />
+    <PackageReference Include="xunit.runner.console" Version="2.3.0-beta5-build3769" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0-beta5-build3769" />
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.0-beta5-build3769" />
```

Comment:
If we're using this CLI tool we should change our Cake scripts to use it (which I think we should. It would simplify our testing harness so we only have to use it instead of custom configs for which frameworks to run the tests on.

Reply:
That was added to make it possible to run unit tests on Mono since `dotnet test` doesn't know how to do that. But currently tests are failing with assertion in mono, so there is no point in changing scripts yet.